### DISCUSSION TITLE
Navigation links with params not becoming active as expected

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#14517] [Help] [answered]
(ver 2.31.6 on cloud)
I think I've found a glitch regarding nav links (both top links and sub links) where:

1. Set a link with params appended, such as **/view?mode=readonly**
2. This link doesn't come active neither when I click that link or visit that url
3. Removing the params part from the link makes it work fine

I think the params part should not be considered in deciding the active link while rendering the navigation area. What do you guys think?

**Comment by ConorWebb96** [comment]
Hey @dannyshin76,

Could you provide some more information on this? I have added params to links and it seems to work fine. Can you confirm the screens you are trying to add params to exist? I've attached a gif below showing this working. If I've maybe misunderstood something please let me know.

![20240905_125742](https://github.com/user-attachments/assets/1f646546-85c2-48f3-b687-5c80f90fbb09)

I've tried this on cloud as well as locally and it works on both instances.

I hope this helps.

**Reply by dannyshin76** [reply]
I'm not saying such links with params doesn't work, I can click them to visit the intended link. However, the clicked link element doesn't get "active" class as expected and stay grayed. Your GIF shows that as well. Seems a minor glitch though. :)

**Reply by ConorWebb96** [reply, marked as answer]
I've create an issue you can view [here](https://github.com/Budibase/budibase/issues/14528). Thanks for clearing that up 🤝
